### ISSUE TITLE
Malformed <div ui-view></ui-view> throws error

### ISSUE THREAD

**Issue by christopherthielen** [angular-ui/ui-router#3205]
```html
<div ui-view></ui-view>
<div ui-view></ui-view>
```
causes:

```
Cannot read property '$$animLeave' of undefined
```

Note in ng 1.5.9 this logs to console but in 1.6.0 it logs "Possibly unhandled error in promise"

## 1.5

http://plnkr.co/edit/2beFjVogMk1zAZXH3vKq?p=preview
```
angular.js:14110 TypeError: Cannot read property '$$animLeave' of undefined
    at VM328 angular-ui-router.js:4024
    at processQueue (angular.js:16606)
    at angular.js:16622
    at Scope.$eval (angular.js:17913)
    at Scope.$digest (angular.js:17727)
    at angular.js:17952
    at completeOutstandingRequest (angular.js:6035)
    at angular.js:6314
```

## 1.6: 

http://plnkr.co/edit/Y5etCjiGk1ehzx7ElYSy?p=preview
```
VM326 angular.js:14324 Possibly unhandled rejection: {}
```

**Comment by christopherthielen** [comment]
I believe this is fixed in 1.0 via https://github.com/angular-ui/ui-router/pull/3164

I will backport this code and see if it fixes the issue in legacy branch.

**Comment by christopherthielen** [comment]
^ Yep, backporting #3164 fixes the issue: http://plnkr.co/edit/GWE5NVnCWBVaksB3Y5KY?p=preview

**Comment by christopherthielen** [comment]
Closed by 5be98e047a096012762096922a2756a0a0a0ea60
